content/github/getting-started-with-github/git-cheatsheet.md

```diff
@@ -6,7 +6,8 @@ versions:
   free-pro-team: '*'
   enterprise-server: '*'
   github-ae: '*'
+intro: Use this git how-to guide to enhance your workflow.  
```

Comment:
In general, when updating conceptual content and intros like this, we try to focus on telling users the "why" behind the feature. For GitHub CLI, here is the high-level value we'd like to tell users about: GitHub CLI allows you to interact with GitHub features without requiring you to switch contexts out of a command line environment. In other words, GitHub CLI allows you to interact with GitHub functionality and features (from PRs, issues, to GitHub Actions) where you are already doing your work.

Can you try writing this intro again and ping me when it's done?